content/en/blogs/faas-openfunction.md

```diff
@@ -0,0 +1,300 @@
+---
+title: 'OpenFunction: Build a Novel Platform for Open-Source Function Computing from Scratch'  
```

Comment:
I prefer `Building a Cloud-Native FaaS Platform on Kubernetes with OpenFunction`. @benjaminhuo How do you think of it?

Reply:
`OpenFunction: Build a modern cloud-native Serverless Computing Platform from scratch` I prefer this

Reply:
How about `OpenFunction: Build a Modern Cloud-Native Serverless Computing Platform on Kubernetes`?

Reply:
cloud native indicates it's running on kubernetes already

Reply:
Simply `OpenFunction: Build a Modern Cloud-Native Serverless Computing Platform` is ok

Reply:
> Simply `OpenFunction: Build a Modern Cloud-Native Serverless Computing Platform` is ok

Updated per your comment.
@FeynmanZhou  Do we need to add `on Kubernetes` for SEO?

Reply:
It's not required to include it in the title since we have used `Cloud-Native` here, but we could add `on Kubernetes` in the subtitles and paragraphs.

Reply:
> It's not required to include it in the title since we have used `Cloud-Native` here, but we could add `on Kubernetes` in the subtitles and paragraphs.

@FeynmanZhou I updated a subtitle in [this commit](https://github.com/kubesphere/website/pull/2249/commits/19fb45953306451bb8e6df1f33d6a4549822e845).